packager/mpd/base/xml/xml_node_unittest.cc

```diff
@@ -354,11 +353,13 @@ TEST(XmlNodeTest, AddAC4AudioInfoMPEGSchemeIMS) {
 class LiveSegmentTimelineTest : public ::testing::Test {
  protected:
   void SetUp() override {
-    FLAGS_segment_template_constant_duration = true;
+    absl::SetFlag(&FLAGS_segment_template_constant_duration, true);
```

Comment:
Same here.  Please use FlagSaver.  Since this is SetUp, you can make it a member variable.